### ISSUE TITLE
WARNING: no real random source present!

### ISSUE THREAD

**Issue by suganthikarthick** [EricssonResearch/openwebrtc#430]
When we run the test client application with custom gstreamer functionality added and we are running it on our target arm architecture. we are getting the following warnings. 

WARNING: no real random source present!"

What does this means?
Thanks.

**Comment by superdump** [comment]
It means that your platform does not appear to have a real source of random data. This perhaps impacts on the encryption parts of WebRTC - that is to say, DTLS. We use OpenSSL for encryption. Does you platform have any good crypto support?

**Comment by suganthikarthick** [comment]
Yes. It does have "libcrypto.so.1.0.0 and libssl.so.1.0.0"  libraries.

**Comment by superdump** [comment]
What platform are you using? Is it known to be supported with standard OpenSSL?

Note that I am not certain this is a real issue but it sounds like it can impact the security of the cryptography algorithms used.

**Comment by sdroege** [comment]
This warning comes from GLib if it can't find a proper randomness source on the target platform.

**Comment by superdump** [comment]
I found information saying that it came from libsrtp. But OK. Perhaps multiple places.

**Comment by suganthikarthick** [comment]
But how to resolve the issue?
Whether it depends on any specific libcrypto library or libsrtp or dtls?
Still we are getting the warnings.

Thanks.
